### PR TITLE
Add small sourmash dbs for testing

### PR DESCRIPTION
**Pull request by elizabethmcd**
This PR fixes the ont samplesheet and adds small sourmash dbs for testing. It feeds in a CSV of each database/lineage pair, which I've made here from E.coli and S.pneumoniae. Having these in separate DBs tests that the workflow is properly taking in all DBs/csvs listed in the input CSV and joining them together to run `gather` and `taxannotate` once for each sample

### CODE REVIEW

File: metagenomics/ont/samplesheet_test.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 sample,fastq_1,fastq_2
 om,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/om_contam_subset_reads.fq.gz,
-el,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/el_contam_subset_reads.fq.gz,
+el,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/el_contam_subset_reads.fq.gz,
```
Comment (taylorreiter) [inline metagenomics/ont/samplesheet_test.csv]: missing new line EOF

File: metagenomics/sourmash_dbs/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# sourmash gather and taxonomy files for test runs
+
+These test files and CSV are used for testing purposes for the Arcadia-Science/metagenomics workflow to ensure the `sourmash gather` and `sourmash taxannoate` modules of the workflow are working properly.
+
+## Generating mini sourmash dbs
+The genomes of E.coli and S.pneumoniae were downloaded and each made into a sourmash dastabase with `sourmash sketch`:
```
Comment (taylorreiter) [inline metagenomics/sourmash_dbs/README.md]:
```suggestion
The genomes of *E. coli* and *S. pneumoniae* were downloaded and each made into a sourmash dastabase with `sourmash sketch`:
```

**Review by taylorreiter — APPROVED** [review]
LGTM!

File: metagenomics/ont/samplesheet_test.csv (hunk 1 above)
Comment (elizabethmcd) [reply metagenomics/ont/samplesheet_test.csv]: fixed

**Pull request merged**